package.json

```diff
@@ -199,6 +198,7 @@
     "stylelint-processor-styled-components": "^1.8.0",
     "supertest": "^4.0.2",
     "swagger-cli": "^2.3.0",
+    "terser-webpack-plugin": "^1.3.0",
```

Comment:
By moving this to dev deps without updating the build pipeline the production build will not be able to use the terser minifier.